### Hi there üëã

<!--
**RemyLau/remylau** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->

I am a Postdoctoral Fellow in the Ray and Stephanie Lane Computational Biology
Department at Carnegie Mellon University. I currently work on developing
computational and machine learning methods, as well as software, for analyzing
and understanding single-cell epigenomics.

I obtained my Ph.D. degree in the Department of Computational Mathematics,
Science & Engineering (CMSE) at Michigan State University. My training focused
on network biology, graph representation learning, spectral graph theory,
and machine learning.

![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=remylau&theme=tokyonight&show_icons=true)

###  üõ† I‚Äôm currently developing
- Stay tuned! Something exciting is going online soon!

### üß∞ I'm actively maintaining several packages related to my past / recent projects
- [obnb](https://github.com/krishnanlab/obnb): a Python toolkit for setting up benchmarking datasets using publicly available biomedical networks and gene annotation resources. A comprehensive benchmarking study with various graph neural networks and graph embedding methods is presented in [obnbench](https://github.com/krishnanlab/obnbench).
- [DANCE](https://github.com/OmicsML/dance): an extensive toolkit for deep learning with single-cell (multi-)omics data.
- [PecanPy](https://github.com/krishnanlab/PecanPy) [[paper](https://academic.oup.com/bioinformatics/article/37/19/3377/6184859)]: a memory efficient and [Numba](https://github.com/numba/numba) accelerated Python implementation of [node2vec](https://snap.stanford.edu/node2vec/) with an improved version [node2vec+](https://github.com/krishnanlab/node2vecplus_benchmarks) [[paper](https://www.biorxiv.org/content/10.1101/2022.08.14.503926v1.abstract)] for weighted graphs. 
- [PyGenePlexus](https://github.com/krishnanlab/PyGenePlexus) [[paper](https://academic.oup.com/bioinformatics/article/36/11/3457/5780279)]: a network-based gene classification service using machine learning and gene interaction network features.
- [GTaxoGym](https://github.com/G-Taxonomy-Workgroup/GTaxoGym) [[paper](https://arxiv.org/abs/2206.07729)]: a taxonomic study of benchmarking graph datasets from various domains based on the GNN model sensitivity to a collection of graph perturbations.

### üì´ Find out more about my work and reach out to me
- [ORCID](https://orcid.org/0000-0002-6025-6492)
- [Google Scholar](https://scholar.google.com/citations?user=by5L-BcAAAAJ)
- [Twitter](https://twitter.com/RemyLau3)
- Email: liurenmi@msu.edu

### ‚ö° Side projects
- ‚úçÔ∏è I share my passion about network biology and machine learning via blog posts on [Medium](https://remylau961.medium.com/)
- üëÄ I create mathematical and algorithmic [visualizations](https://github.com/RemyLau/visualizations) using [Manim](https://github.com/ManimCommunity/manim), which was first developed and used by my favorite math YouTube channel [3Blue1Brown](https://www.3blue1brown.com/).
- ü§ó I contribute to open source projects in various ways
  - Implemented patches and fixes: [Pytorch Geometric](https://github.com/pyg-team/pytorch_geometric)
  - Feature discussion: [Hydra](https://github.com/facebookresearch/hydra)
  - Bug report and question posting: [mypy](https://github.com/python/mypy), [mydisease.info](https://github.com/biothings/mydisease.info), [ndex2-client](https://github.com/ndexbio/ndex2-client)
- üì¶ I work on several small packages on the side to help improve my production workflow and exercise my dev workflow
  - [pydab](https://github.com/krishnanlab/pydab): a tool for working with [dab](https://functionlab.github.io/sleipnir-docs/Dat2Dab.html) files used by [Sleipnir](https://functionlab.github.io/sleipnir-docs/), a C++ library for machine learning on genomic data.
  - [py2zenodo](https://github.com/RemyLau/py2zenodo): a command line tool for uploading data to [Zenodo](https://zenodo.org/)
  - [fastauroc](https://github.com/RemyLau/fastauc): a [Numba](https://github.com/numba/numba) accelerated computation of the area under the receiver operating characteristic.

### Installation notes

```bash
conda create -n remylau python=3.11 -y && conda activate remylau

pip install -e .

conda clean --all -y
```
